### PR TITLE
Pipes change append to extend

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: modules/angular2/test/change_detection/pipes/pipes_spec.ts [hunk 1]
```diff
@@ -76,13 +76,47 @@ export function main() {
...
+    describe('.create()', () => {
+      it("should create a new Pipes object", () => {
+        firstPipeFactory.spy("supports").andReturn(true);
+        firstPipeFactory.spy("create").andReturn(firstPipe);
+
+        var pipes = Pipes.create({'async': [firstPipeFactory]});
+        expect(pipes.get("async", "first")).toBe(firstPipe);
+      });
+
+      it("should prepend passed it config to existing registry", () => {
```
Comment (jeffbcross) [inline modules/angular2/test/change_detection/pipes/pipes_spec.ts]: Wording here doesn't make sense.

File: modules/angular2/src/change_detection/pipes/pipes.ts [hunk 2]
```diff
@@ -67,37 +67,35 @@ export class Pipes {
    * })
    * ```
    */
-  static append(config): Binding {
+  static extend(config): Binding {
     return new Binding(Pipes, {
       toFactory: (pipes: Pipes) => {
-        if (!isPresent(pipes)) {
+        if (isBlank(pipes)) {
           // Typically would occur when calling Pipe.append inside of dependencies passed to
```
Comment (gdi2290) [inline modules/angular2/src/change_detection/pipes/pipes.ts]: Pipe.append -> Pipe.extend

**Comment by jeffbcross** [comment]
LGTM with one typo fix. Is there no open issue corresponding to this?

**Comment by jeffbcross** [comment]
Ah, and @gdi2290 pointed out comments that should be updated. Do a quick search and update those before merging plz.

**Pull request merged**